src/Faker/Calculator/Inn.php

```diff
@@ -15,8 +15,11 @@ class Inn
     public static function checksum($inn)
```

Comment:
There is possible add INN type (Person or Company) and then use it in cases inside method